TWLight/applications/templates/applications/request_for_application.html

```diff
@@ -19,6 +19,12 @@
 
 {% comment %}Translators: We are asking editors which publishers' resources they want access to. They may request more than one publisher.{% endcomment %}
 <h1>{% trans 'What resources do you want to access?' %}</h1>
+ <div class="well well-sm">
+      {% blocktrans trimmed %}
```

Comment:
You should add a comment to help translators translate this text. Add this before  `blocktrans`

```
{% comment %}Translators: This text explains that Library Bundle partners are not shown in this list.{% endcomment %}
```